src/content/api/native-events.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: Centrapay Native Events
+description: Events that are native to Centrapay
+nav:
+  path: API/Events
+  order: 2
+---
+
+The following table describes the types of events that are native to Centrapay. No actions is required as the native events are handled by Centrapay.
+
+
+|      Event Type       |                            Category                            |
+| :-------------------- | :------------------------------------------------------------- |
+| `refund:completed`    | [Payment Request](/api/payment-requests#payment-request-model) |
+| `promotion:completed` | [Loyalty Program](/api/loyalty-programs#loyalty-program-model) |
+| `promotion:updated`   | [Loyalty Program](/api/loyalty-programs#loyalty-program-model) |
```

Comment:
Where did you get this list from? I don't think this is correct. The only one we are handling internally now is `payment-request:paid`

Reply:
We should also mention that integrators should NOT send these event types to our system

Reply:
> Where did you get this list from? I don't think this is correct. The only one we are handling internally now is `payment-request:paid`

@Ray2126  That came from the Description in API design doc https://www.notion.so/centrapay/Create-Event-91b0c6b2050f42e29cc1730d2a746880?pvs=4

Reply:
> We should also mention that integrators should NOT send these event types to our system

@Ray2126  Yep will add it and put the info into a guide. Keeping the API doc conscise